docs/UsersGuide/source/BuildingRunningTesting/BuildSRW.rst

```diff
@@ -277,96 +267,16 @@ After running ``devbuild.sh``, the executables listed in :numref:`Table %s <Exec
-   | process_updatesst.exe      | Updates Sea Surface Temperature (SST) field based on the SST analysis from  |
-   |                            | NCEP.                                                                       |
-   +----------------------------+-----------------------------------------------------------------------------+
-   | ref2tten.exe               | Calculates temperature tendency based on the radar reflectivity observation |
-   |                            | at each grid point. This temperature tendency can be used by the model      |
-   |                            | during integration as latent heating initialization for ongoing             |
-   |                            | precipitation systems, especially convection.                               |
-   +----------------------------+-----------------------------------------------------------------------------+
-   | test_nc_unlimdims.x        | Checks to see the number of fields with unlimited dimensions in NetCDF      |
-   |                            | files.                                                                      |
-   +----------------------------+-----------------------------------------------------------------------------+
-   | ufs_srweather_app.settings |                                                                             |
-   +----------------------------+-----------------------------------------------------------------------------+
-   | update_bc.exe              | Adjusts 0-h boundary conditions based on the analysis results during data   |
-   |                            | assimilation cycling.                                                       |
-   +----------------------------+-----------------------------------------------------------------------------+
-   | update_GVF.exe             | Updates the GVF in the surface file based on the real-time observation      |
-   |                            | files.                                                                      |
-   +----------------------------+-----------------------------------------------------------------------------+
-   | update_ice.exe             | Replaces ice fields in warm start surface files based on the forecast from  |
-   |                            | cold start forecast using the GFS as the initial file.                      |
-   +----------------------------+-----------------------------------------------------------------------------+
-   | use_raphrrr_sfc.exe        | Uses RAP and HRRR surface fields to replace the surface fields in FV3LAM.   |
-   |                            | This is only used for starting the RRFS surface cycling.                    |
-   +----------------------------+-----------------------------------------------------------------------------+
+   | weight_gen             | Creates ESMF ``scrip`` files for gaussian grids. These NetCDF-formatted files   |
```

Comment:
`weight_gen` is one of the (newer) UFS_UTILS utilities, and it looks like 'scrip' files are referencing ESMF "Spherical Coordinate Remapping and Interpolation Package (SCRIP)" files. I'll capitalize to avoid confusion. I think it's a relatively new utility, and while I don't think it's actively used by the SRW App, the executable does appear in the `exec` folder. Seems strange to define some executables but not others.